app/controllers/needs_controller.rb

```diff
@@ -74,6 +71,16 @@ def set_contact
   end
 
   def need_params
-    params.require(:need).permit(:name, :status, :user_id, :role_id, :category, :is_urgent, :lock_version)
+    params.require(:need).permit(:name, :status, :assigned_to, :category, :is_urgent, :lock_version)
+  end
+
+  def construct_assigned_to_options
```

Comment:
as this is duplicate code I wonder if it's better placed in a view helper rather than the controller?